io/plugins/eu.esdihumboldt.hale.io.csv.ui/src/eu/esdihumboldt/hale/io/csv/ui/TypeSelectionPage.java

```diff
@@ -202,11 +202,11 @@ public boolean updateConfiguration(InstanceReader provider) {
 				Value.of(skipNlinesSpinner.getSelection()));
 
 		if (customFormat.isVisible()) {
-			provider.setParameter(ReaderSettings.PARAMETER_DATE_FORMAT,
-					Value.of(customFormat.getText()));
+			provider.setParameter(CSVConstants.PARAMETER_DATE_FORMAT, Value.of(customFormat
+					.getText().replace("mm", "MM").replace("DD", "dd").replace("YYYY", "yyyy")));
```

Comment:
Why are minutes replaced by months?

Reply:
@stempler Good question. That come from Kate that has been adding date as **dd.mm-yyyy** as a `custom format`.
I would then propose to remove this `replace `and add in the interface a text informing on how to custom format should be composed.